plugins/resources/config/routes.rb

```diff
@@ -5,33 +5,8 @@
   # NOTE: "current" used to be ":cluster_id". Since multi-cluster support has
   # been removed from the UI, the value "current" is hardcoded to preserve
   # backwards compatibility.
-
-  get "/project/current/:override_domain_id/:override_project_id" =>
-        "application#project",
-      :as => "foreign_project"
-  get "/project" => "application#project", :as => "project"
-  get "/project/bigvm_resources" => "application#bigvm_resources",
-  :as => "bigvm_resources"
```

Comment:
This is used by the new UI and needs to stay.

Reply:
That was an oversight. Thank you.